src/app.py

```diff
@@ -829,6 +864,22 @@ def publish_datastage(identifier):
                     return jsonify({"error": f"{dataset_uuid} missing contacts or contributors. Must have at least one of each"}), 400
             ingest_helper = IngestFileHelper(app.config)
 
+            # Save a .json file with the metadata information at the top level directory...
+            if dataset_ingest_matadata_dict is not None:
+                json_object = json.dumps(dataset_ingest_matadata_dict['metadata'], indent=4)
+                json_object += '\n'
+                ds_path = ingest_helper.dataset_directory_absolute_path(dataset_data_access_level,
```

Comment:
The dataset directory path doesn't seem to be right. We don't use the `dataset_group_uuid` directory, do we? @shirey Should it be the dataset uuid directory instead? 

For example, `/hive/hubmap-dev/data/consortium/IEC Testing Group/5435d6a861c0ac56df9a7f3d092fa679` where `IEC Testing Group` is the group name, not uuid. And `5435d6a861c0ac56df9a7f3d092fa679` is the dataset uuid.

Reply:
Joe, I'll have to defer to Bill on this one.

Reply:
@shirey I did test this locally, and it seemed to write the file in what I think was the right directory locally.

DBMI-C02DX03UMD6R:07a29e4c-ed43-11e8-b56a-0e8017bdda58 cpk36$ pwd
/Users/cpk36/hive/hubmap-dev/data/consortium/IEC Testing Group/07a29e4c-ed43-11e8-b56a-0e8017bdda58
DBMI-C02DX03UMD6R:07a29e4c-ed43-11e8-b56a-0e8017bdda58 cpk36$ ls -al
total 8
drwxr-xr-x  3 cpk36  731655015    96 Sep 26 16:15 .
drwxr-xr-x  3 cpk36  731655015    96 Oct  2 13:47 ..
-rw-r--r--@ 1 cpk36  731655015  1304 Oct  2 13:50 metadata.json

This is an example.

Reply:
@ChuckKollar That test looks incorrect because the `uuid` at the end of the path is not a dataset id, rather looks like the group uuid?